sal_interfaces/OCPS/OCPS_Events.xml

```diff
@@ -28,4 +28,31 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>OCPS</Subsystem>
+    <EFDB_Topic>OCPS_logevent_job_id</EFDB_Topic>
```

Comment:
I have a preference for this being called: OCPS_logevent_jobId.

Reply:
The name `job_id` was for parallelism with the existing `job_result`.